src/plugins/UtilCommands/controllers/commands/getWarnings.js

```diff
@@ -0,0 +1,20 @@
+const { User, Warning } = require("./../../../../../api/models/index.js");
+
+exports.help = {
+  name: "getWarnings",
+  description: "Creates a custom thread invite.",
+  usage: "addThreadCustomInvite <id> <emoji>",
+  example: "addThreadCustomInvite 815606090183606323 👍",
+};
+
+exports.conf = {
+  aliases: ["addcustom"],
+  cooldown: 5,
+  permissions: ["ADMINISTRATOR"],
+  arguments: [],
+};
+
+exports.run = async (client, message, args) => {
+  const warnings = await Warning.findAll({ where: { userId: message.author.id } });
+  console.log(warnings);
```

Comment:
What is the goal of this function? Is it to log something in the server?

Reply:
I made this command to test how I would be able to retrieve the warnings a user had. It's in the UtilCommands plugin, and the command will not be there once this is pushed to production.